### DISCUSSION TITLE
Curriculum studente

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#543] [Idee e proposte]
Buonasera ,il preside mi ha appena fatto notare che tutti i registri presentano una sezione chiamata "curriculum",che non ha nulla a che fare con il famoso curriculum dello studente che conosciamo ,ma  è semplicemente una sezione del registro che riporta gli anni scolastici precedenti e gli esiti finali accompagnati dalla media ,vi faccio un piccolo esempio:

Anno scolastico 2022/2023  Esito: "Ammesso alla classe successiva" Media:9.00

Anno scolastico 2021/2022  Esito: "Ammesso alla classe successiva" Media:9.40

Anno scolastico 2020/2021  Esito: "Ammesso alla classe successiva" Media:8.40

**Comment by trinko** [comment]
Lo storico degli esiti non è previsto.
Lo segno comunque come richiesta.
